src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.html

```diff
@@ -7,7 +7,11 @@
 
 <span
     class="dot-persona-selector-option__label"
-    [ngClass]="{ 'dot-persona-selector-option__personalized': persona.personalized }"
+    style="-webkit-box-orient: vertical;-moz-box-orient: vertical;"
+    [ngClass]="{
+        'dot-persona-selector-option__personalized': persona.personalized,
```

Comment:
I think we don't have test for this

Reply:
??

Reply:
https://github.com/dotCMS/core-web/blob/9b19164220b2cc77ffd525aeb5629dcd79a07dbf/src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.spec.ts#L49